rivanna/README.md

```diff
@@ -0,0 +1,183 @@
+# Running Cylon on Rivanna
+
+Arup Sarker (arupcsedu@gmail.com, djy8hg@virginia.edu)
+
+## Issues on Summit
+
+Running cylon on rivanna requires a special configuration. We have verified
+that installations based on conda with virtual environment
+
+The solution is to use the guide we provide here.
+
+## Intsall instructions
+
+Clone the repository
+
+```shell
+git clone https://github.com/cylondata/cylon.git
+cd cylon
+```
+
+There are two ways you can build cylon on Rivanna
+
+### Fix all dependencies for Cylon on Rivanna without the dependencies of the system gcc and openmpi version.
+
+
+
+1. Change in yml for conda build: conda/environments/cylon.yml
+Add this libraries
+  - gcc
+  - gxx
+  - gxx_linux-64
+
+
+```shell
+name: cylon_rivanna
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - python=3.10
+  - cmake>=3.23.1,!=3.25.0
+  - arrow-cpp=9
+  - pyarrow=9.0.0
+  - glog
+  - openmpi=4.1.3=ha1ae619_105
+  - ucx>=1.12.1
+  - cython>=0.29,<0.30
+  - numpy
+  - pandas>=1.0,<1.6.0dev0
+  - fsspec>=0.6.0
+  - setuptools
+  # they are not needed for using pygcylon or compiling it
+  - pytest
+  - pytest-mpi
+  - mpi4py
+  - gcc
+  - gxx
+  - gxx_linux-64
+  
+ ```
+
+Added all blue packages to overcome the dependencies on the preloaded modules.
+
+```shell
+conda env create -f conda/environments/cylon.yml

```

Comment:
Should we specify a `Python` version for this?

Reply:
We don't need it. Rivanna's default python version works perfectly.